lib/elasticsearch.js

```diff
@@ -44,16 +45,17 @@ var elasticCertCa;
 
 var elasticStats = {};
 
-function transform(stats, index, type) {
+function transform(stats, index, statsdType, esType) {
   const payload = [];
 
   for (let i = 0; i < stats.length; i++) {
     payload.push(JSON.stringify({
       index: {
         "_index": index,
-        "_type": type
+        "_type": esType
       }
     }));
+    stats[i].type = statsdType;
```

Comment:
I think I still like having this original type accessible as a reference so you can tell which statsd type the current stat is using.